pkg/downstream/downstream.go

```diff
@@ -60,6 +60,20 @@ func SetDownstreamVersionPendingPreflight(appID string, sequence int64) error {
 	return nil
 }
 
+// GetDownstreamVersionStatus gets the status for the downstream version with the given sequence and app id
+func GetDownstreamVersionStatus(appID string, sequence int64) (string, error) {
+	db := persistence.MustGetPGSession()
+	query := `select status from app_downstream_version where app_id = $1 and sequence = $2`
```

Comment:
i think this should be parent_sequence?

Reply:
all the others are using sequence, I'm afraid

Reply:
see https://github.com/replicatedhq/kotsadm/pull/1272/files#diff-d1939c52afd5d08855269da22c717483R54, https://github.com/replicatedhq/kotsadm/pull/1272/files#diff-d1939c52afd5d08855269da22c717483L42